content/sdk/10.js/00.ethers/00.ethers.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Getting Started"
+menuTitle: "JavaScript Versions"
+description: "Explore zkSync-ethers SDK versions"
+tags: "zksync, sdks, zkSync Era SDK, ethereum, era network, javascript versions"
+---
+
+These versions help developers interact with the zkSync Era blockchain. It covers most common
+operations developers require and provides a starting point for those new to the zkSync Era.
+
+::callout
+Take a look at [migration guide](/sdk/js/ethers/v6/migration) if you are migrating from `zksync-web3`.
+::
+
+### Prerequisites
+
+- Node: >=v18 ([installation guide](https://nodejs.org/en/download/package-manager))
+
+### Installation
+
+To install SDK for zkSync Era, run the command below in your terminal.
+
+::code-group
+
+```bash [ethers-v5]
+yarn add zksync-ethers@5
+yarn add ethers@5 # ethers is a peer dependency of zksync-ethers
+```
+
+```bash [ethers v6]
+yarn add zksync-ethers
+yarn add ethers@6 # ethers is a peer dependency of zksync-ethers
+```
+
+::
+
+For more detailed information, learn more!
+
+::card-group
+::card
+---
+title: V5
+to: /sdk/js/ethers/v5/getting-started
+target: _blank
+---
+Integrate JS with zkSync Era for secure transactions.
+::
+::card
+---
+title: V6
+to: /sdk/js/ethers/v6/getting-started
+target: _blank
```

Comment:
Same for these cards, all internal links should not have a `target: _blank`

Reply:
done